lvgl_helpers.c

```diff
@@ -105,8 +105,6 @@ void lvgl_driver_init(void)
         DISP_SPI_IO2, DISP_SPI_IO3);
 
     disp_spi_add_device(TFT_SPI_HOST);
-
-    disp_driver_init();
```

Comment:
Is this delete just becouse signature of `disp_driver_init` has changed and the code wasn't updated?

Reply:
We're initializing the display driver in main, so `lvgl_driver_init` will only initialize the peripherals, we may need to change it's signature.

Reply:
In that case, all calls to `disp_driver_init` should be deleted from this function, shouldn't they?

Reply:
Yes, they need to be deleted, I will do so later today.

Reply:
Updated in 47756f4